src/baldr/openlr.cc

```diff
@@ -1,97 +0,0 @@
-#include <cmath>
-#include <string>
-#include <vector>
-
-#include "baldr/graphconstants.h"
-#include "baldr/openlr.h"
-#include "midgard/encoded.h"
-#include "midgard/pointll.h"
-
-#include "proto/trip.pb.h"
-#include "proto/tripcommon.pb.h"
-
-namespace valhalla {
-namespace midgard {
-
-using FormOfWay = OpenLR::LocationReferencePoint::FormOfWay;
-
-FormOfWay road_class_to_fow(const TripLeg::Node& node) {
-  const bool is_missing_rc = !node.has_edge() || !node.edge().has_road_class();
-  if (is_missing_rc || !node.edge().has_use()) {
```

Comment:
this shouldnt happen. we should never make an openlr without an edge to make it from and every edge will always have a road class.